### ISSUE TITLE
why is the doc object not alway available in custom renderSuggestion templates?

### ISSUE THREAD

**Issue by illmat** [matteodem/meteor-easy-search#204]
Hey, 
what's the advantage of not always returning the doc object for the current suggestion?

from your code:
```javascript
// Not found in the first field
if (parts.length === 1) {
  return {
    'pre': firstFieldValue,
    'value': firstFieldValue,
    'id': this._id,
    'options': options
  };
}

return {
  'pre': parts[0],
  'found': (new RegExp(searchValue, 'i')).exec(firstFieldValue).shift(),
  'post': parts[1],
  'value': firstFieldValue,
  'id': this._id,
  'doc': this,
  'options': options
};
```

If I want to search over multiple fields, like lastname and firstname in this case:
```javascript
EasySearch.createSearchIndex('contacts', {
  'field': ['lastname', 'firstname']
});
```
```html
<template name="suggestionTpl">
  {{doc.firstname}} {{pre}}<span class="found">{{found}}</span>{{post}}
</template>
```
The missing firstname in the suggestion list just looks stupid if there are mixed results.
Maybe you could build one big configurable string and use your pre/found/post logic then.

Keep up the good work!
illmat

**Comment by matteodem** [comment]
The PR for this should be fairly easy. If you like you can submit a PR or I'll do it in the next days.

**Comment by matteodem** [comment]
This should be fixed as of the latest version 1.5.7, please write into here if the problem still exists.
